mautic/mautic
# Salesforce sandbox

Please answer the following questions:

| Q             | A
| ------------- | ---
| Bug fix?      | n
| New feature?  | y
| BC breaks?    | n
| Deprecations? | n
| Fixed issues  |  n

## Description
This PR adds Sandbox option to the salesforce plugin

## Steps to test this PR
Apply this PR in the configuration window for the salesforce plugin you will be able to see the sandbox feature under the features tab. If selected, Mautic will post to Salesforce sandbox, if unselected Mautic will post to Salesforce production
